.github/workflows/build.yml

```diff
@@ -876,10 +876,8 @@ jobs:
       matrix:
         node: [18, 20, 22]
         remix: [1, 2]
-        # Remix v2 only supports Node 18+, so run Node 14, 16 tests separately
+        # Remix v2 only supports Node 18+, so run 16 tests separately
         include:
-          - node: 14
```

Comment:
Why did we drop this here? Is this not supported by otel?

Reply:
This is not supported by vitest. (vitest says node 16 is also not supported but it runs fine here)